tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -59,324 +63,321 @@ create_boundary_condition() {
-                        Direction<3>::lower_zeta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_xi(), Direction<3>::upper_zeta(),
-                        Direction<3>::lower_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_xi(), Direction<3>::lower_zeta(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_zeta(), Direction<3>::upper_xi(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::lower_zeta(), Direction<3>::lower_xi(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map});
-    if (inner_cube_sphericity == 0.0) {
-      if (use_equiangular_map) {
-        local_coord_maps.emplace_back(
-            make_coordinate_map_base<Frame::BlockLogical, TargetFrame>(
-                Equiangular3D{
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0)),
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0)),
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0))}));
+  const auto& blocks = domain.blocks();
+  const auto block_names = sphere.block_names();
+  const size_t num_blocks = block_names.size();
+  const auto all_boundary_conditions = sphere.external_boundary_conditions();
+  double current_time = 1.0;
+
+  MAKE_GENERATOR(gen);
+
+  for (size_t block_id = 0; block_id < num_blocks; ++block_id) {
+    const auto& block = blocks[block_id];
+    const auto& boundary_conditions = all_boundary_conditions[block_id];
+
+    // verify if adjacent to inner block
+    const auto abuts_inner_block =
+        [&num_blocks](const auto& direction_and_neighbor) {
+          const auto& [direction, neighbor] = direction_and_neighbor;
+          (void)direction;
+          if (neighbor.id() == (num_blocks - 1)) {
+            return true;
+          } else {
+            return false;
+          }
+        };
+
+    tnsr::I<double, 3, Frame::BlockLogical> block_logical_coords =
```

Comment:
Maybe name something like `coords_on_spherical_partition`? Also `const auto`

Please explain a bit what you're testing in comments.